textx/model.py

```diff
@@ -843,11 +903,8 @@ def call_obj_processors(metamodel, model_obj,
 
                     # We have model loaded and all link resolved
                     # So we shall do a depth-first call of object
-                    # processors if any processor is defined.
-                    if m._tx_metamodel.obj_processors:
-                        if parser.debug:
-                            parser.dprint("CALLING OBJECT PROCESSORS")
-                        call_obj_processors(m._tx_metamodel, m)
+                    # processors
+                    call_obj_processors(m._tx_metamodel, m)
```

Comment:
This `dprint` should probably be now moved to `call_obj_processors`

Reply:
I added it outside the `call_obj_processors` (as before), since `call_obj_processors` is called recusively.

Reply:
(code reactivated / reverted deletion)